epwing.go

```diff
@@ -78,14 +78,17 @@ func epwingExportDb(inputPath, outputPath, language, title string, stride int, p
 
 	var data []byte
 	if toolExec {
+		ex, err := os.Executable()
+		if err != nil {
+			panic(err)
```

Comment:
Why panic? This will kill the UI and the user will probably not know what is going on.